src/CHANGES.txt

```diff
@@ -6,8 +6,14 @@
 
 RELEASE  VERSION/DATE TO BE FILLED IN LATER
 
-  From Mathew Robinson:
+  From Jason Kenny
+    - Fix python3 crash when Value node get_text_content when child content does not have decode()
+  
+  From Peter Diener:
```

Comment:
Hmm. why are there changes here which aren't yours?

Reply:
Does he need to rebase??